src/components/experiment-wrapper/experiment-wrapper.js

```diff
@@ -181,7 +181,7 @@ const ExperimentWrapper = ({ theme }) => {
             rel="noreferrer"
             target="_blank"
           >
-            <Button theme={theme}>View docs</Button>
```

Comment:
I noticed you're removing the `theme` prop here, which is fine, but didn't remove it on the buttons in the `src/components/export-modal/export-modal.js` file. I wonder why? I think you can remove it there as well can't you?

Reply:
yes, i will remove it :) .. thanks!